content/soarian/dstu2/general-clinical/procedure.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Procedure | Soarian DSTU 2 API
+---
+ 
+# Procedure
+
+* TOC
+{:toc}
+
+## Overview
+
+The API returns procedures charted in Soarian Clinicals. Procedures in completed, aborted, or erroneous status are not returned. Note that traditional charting practices have procedures resolved and charted in coding and billing systems and not interfaced back to the EMR; therefore, the procedure record in Soarian may not be complete nor precise to a procedure list provided as part of a patient's billing record.  
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_procedure, :dstu2) %>
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+## Search
+
+    GET /Procedure?:parameters
+    
+_Implementation Notes_
```

Comment:
Added an implementation note as 'Procedures in erroneous state are not returned.'